views/forms/intake_packet.php

```diff
@@ -747,222 +747,299 @@
                                         <div class="col-sm-4">
                                             <label class="form-control-label">Do you now, or have you ever had a problem with drug/alcohol abuse?</label>
                                         </div>
-                                        <label class="form-check-label" for="drug_alcohol_abuse_yes">
-                                            <input type="radio" id="drug_alcohol_abuse_yes" name="drug_alcohol_abuse" value="Yes"> Yes
+                                        <label class="custom-control custom-radio" for="drug_alcohol_abuse_yes">
+                                            <input type="radio" id="drug_alcohol_abuse_yes" name="drug_alcohol_abuse" class="custom-control-input" value="Yes">
+                                            <span class="custom-control-indicator"></span>
+                                            <span class="custom-control-description">Yes</span>
                                         </label>
-                                        <label class="form-check-label" for="drug_alcohol_abuse_no">
-                                            <input type="radio" id="drug_alcohol_abuse_no" name="drug_alcohol_abuse" value="No"> No
+                                        <label class="custom-control custom-radio" for="drug_alcohol_abuse_no">
+                                            <input type="radio" id="drug_alcohol_abuse_no" name="drug_alcohol_abuse" class="custom-control-input" value="No">
+                                            <span class="custom-control-indicator"></span>
+                                            <span class="custom-control-description">No</span>
                                         </label>
                                     </div>
+                                    <!-- End Q: Drug/Alcohol Abuse -->
 
-                                    <div class="form-group hidden-field row drug_alcohol_abuse_div answer_yes">
+                                    <div class="form-group hidden-field row drug_alcohol_abuse_div_yes answer_yes">
```

Comment:
The label on line 764 is "Please explain:" and the placeholder is also "Please explain...". In this case I don't think we need the placeholder.

Reply:
Fixed all placeholder issues